### ISSUE TITLE
ensure real error is returned when doing multi-asset scan 

### ISSUE THREAD

**Issue by vjeffrey** [mondoohq/cnquery#1848]
we had an issue with this in v8, 
https://github.com/mondoohq/cnquery/blob/v8/explorer/scan/local_scanner.go#L167

**Comment by czunker** [comment]
This works diffently now and looks good to me:
https://github.com/mondoohq/cnquery/blob/main/explorer/scan/local_scanner.go#L186-L192

**Comment by vjeffrey** [comment]
oh ya, there's actually a comment about that down below:
```
// TODO: we want to keep better track of errors, since there may be
		// multiple assets coming in. It's annoying to abort the scan if we get one
		// error at this stage.
```

**Comment by chris-rock** [comment]
fixed by #3152
